### PR TITLE
Create a microservice that accepts connections from the substrate node.

### PR DESCRIPTION
**Pull request by mgralinski-bright**

### CODE REVIEW

File: bin/authority-service/src/config.rs [hunk 1]
```diff
@@ -0,0 +1,77 @@
+use serde::Deserialize;
+use serde_json::Result;
+use std::fs;
+
+#[derive(Deserialize)]
+pub struct Config {
+	nodes: Vec<String>,
```
Comment (kziemianek) [inline bin/authority-service/src/config.rs]: Why do we need this field ?

**Review by kziemianek — CHANGES_REQUESTED** [review]
There are couple of compiler's warning, please take a look at them.
If this binary need to be built with nightly please update README or make it build with stable release channel

File: bin/authority-service/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* node name authorization:
+
+## Build
+```
+cargo build -p authority-service
+```
+
+## Run
+```
+./target/debug/authority-service <path-to-config-file.json>
```
Comment (kziemianek) [inline bin/authority-service/README.md]: Seem's like this command doesn't work

```
% ./target/debug/authority-service config/config.json
zsh: no such file or directory: ./target/debug/authority-service
```

Maybe just  `cargo run -- config/config.json` ?

**Review by kziemianek — CHANGES_REQUESTED** [review]

File: bin/authority-service/README.md (hunk 2 above)
Comment (mgralinski-bright) [reply bin/authority-service/README.md]: I've changed this to run optional without configuration file.

File: bin/authority-service/src/config.rs (hunk 1 above)
Comment (mgralinski-bright) [reply bin/authority-service/src/config.rs]: Removed

File: bin/authority-service/README.md (hunk 2 above)
Comment (kziemianek) [reply bin/authority-service/README.md]: Error says there is no binary, not configuration file 🙃

Comment (kziemianek) [reply bin/authority-service/README.md]: `cargo +nightly run` works for me

**Review by kziemianek — APPROVED** [review]

**Pull request merged**